src/components/calculator.css

```diff
@@ -0,0 +1,40 @@
+.Container {
+  border: #000 solid 2px;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: repeat(5, 1fr);
+  width: 40%;
+  margin: 5% 30%;
+}
+
+.Screen {
+  background-color: #858693;
+  text-align: right;
+  font-size: 30px;
+  color: #fff;
+}
+
+.orange {
+  background-color: #f5913f;
+}
+
+.white {
+  color: #fff;
+}
+
+.btn-style {
+  font-size: 1.5rem;
+  padding: 30px 0;
+  padding: 20px;
+  text-align: center;
+  width: 25%;
+  color: #000;
+}
+
+.column {
+  width: 100%;
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235907074-de6eb3b6-ff08-4f4d-9a0a-19e7b6100834.png)

**[OPTIONAL]**

- [ ] It would be nice if you put `npm start` command in the section shown in the image above :arrow_up: to make your readme descriptive and professional :blush: